tests/test_definitions/test_expectations_cfe.py

```diff
@@ -96,7 +112,9 @@ def pytest_generate_tests(metafunc):
                             # if we're not on the "only_for" list, then never even generate the test
                             generate_test = False
                             if not isinstance(test["only_for"], list):
-                                raise ValueError("Invalid test specification.")
+                                raise ValueError(
+                                    "Invalid test specification. 'only_for' should be passed in as a list"
```

Comment:
This one kindof bit me during development :) adding some more detail in the error message